DocUI/index.ts

```diff
@@ -0,0 +1,27 @@
+import { promises as fs } from "fs"
+
+import { AzureFunction, Context } from "@azure/functions"
+import { getAbsoluteFSPath } from "swagger-ui-dist"
+
+const swaggerJsdocUI: AzureFunction = async function (context: Context): Promise<void> {
+  let filename = context.bindingData.filename
```

Comment:
don't use `let`. if a `filename` is not specified (i.e. if a user navigates to `/api/docs`), they should be redirected to `/api/docs/index.html` with a 308 redirect.

to do this, you need to set the status code to 308, and the `location` header to the path you want to redirect them to. do this if and only if a `filename` is not specified.